app/models/article.rb

```diff
@@ -0,0 +1,13 @@
+class Article < ApplicationRecord
+  include Visible
+
+  belongs_to :owner, class_name: 'User'
+  has_many :comments, dependent: :destroy
+
+  validates :title, presence: true
+  validates :body, presence: true, length: { minimum: 10 }
+
+  # Handle of dont permit that the article dont get the same title
+  validates :title, uniqueness: {case_insensitive: false}
```

Comment:
@sansuaza how model validations work?

Reply:
the validations take the attribute that is indicated, then the necessary validations are indicated in this field, such as the minimum length, if is required its presence or if it cannot be repeated